store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -11474,4 +11475,43 @@ public AddressList getAddressList(String zimbraId) throws ServiceException {
         }
         return list;
     }
+
+    @Override
+    public void modifyAddressList(String addressListId, Domain domain, String name, String desc,
+        Map<String, String> attrs) throws ServiceException {
```

Comment:
Stephan, can you check if only prov.modifyAttrs woill no server the purpose. Do we need to have this method

Reply:
I can do all of this logic in the document handler, but looking at the other modify methods in this class - this seemed the appropriate place to use zlc.

Reply:
Also worth mentioning, prov.modifyAttrs tries and fails to cast AddressList to an ldap entry so would need to make some small changes.